src/url-replacements.js

```diff
@@ -239,6 +242,14 @@ class UrlReplacements {
       return this.getTimingData_('navigationStart',
           'domContentLoadedEventStart');
     });
+
+    // Returns an identifier for the viewer.
+    this.set_('VIEWER', () => {
+      const viewerUrl = viewerFor(this.win_).getParam(VIEWPORT_ORIGIN_PARAM_) ||
```

Comment:
ok.. so remove the referrer part from calculation of viewer origin? In that case, how do we ensure that origin is always specified?